ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/TestCaseTest.java

```diff
@@ -234,9 +230,8 @@ public void shouldFailForInvalidToplogy() {
   private TopologyTestDriverContainer getSampleTopologyTestDriverContainer() {
     return TopologyTestDriverContainer.of(
         topologyTestDriver,
-        ImmutableList.of(new KsqlTopic("FOO", "foo_kafka", new KsqlJsonSerdeFactory(), false)),
-        new KsqlTopic("BAR", "bar_kafka", new KsqlJsonSerdeFactory(), false),
-        new Pair<>(WindowType.NO_WINDOW, Long.MIN_VALUE)
+        ImmutableList.of(new Topic("FOO", Optional.empty(), new StringSerdeSupplier(), 1, 1)),
```

Comment:
any reason not to use `topic` here?

Reply:
Yes, topic was used as input topic for the TestCase so I didn't want to reuse it here.